br/pkg/lightning/restore/restore.go

```diff
@@ -1667,6 +1667,10 @@ func (rc *Controller) enforceDiskQuota(ctx context.Context) {
 }
 
 func (rc *Controller) setGlobalVariables(ctx context.Context) error {
+	// skip for tidb backend to be compatible with MySQL
+	if rc.isTiDBBackend() {
```

Comment:
Could we rename `BackendTiDB` to `BackendSQL` ?

Reply:
`BackendSQL` should be a better name. Not sure if we should also change the corresponding config fields. @kennytm what do you think?

Reply:
yes, but we could do it in another PR.

`backend = "tidb"` in the config should still be accepted (as alias of `backend = "sql"`).